packages/cf-component-loading/src/Loading.js

```diff
@@ -1,8 +1,45 @@
 import React from 'react';
-import Icon from 'cf-component-icon';
+import { Icon } from 'cf-component-icon';
+
+import { createComponent } from 'cf-style-container';
+
+const LoadingWrapper = createComponent(({}) => ({
+  width: 40,
```

Comment:
This seems wider than our previous default. Have you tried implementing this yet and seeing how it looks?

Reply:
Aiye. My bad. Forgot I had this left to do. So Nena is going to give me some new animation values. Which I will swap out. But the loading icon will have 
3 colors - default,white, orange
All sizes allowed by cf-component-icon
3 types -  icon, loading text and opacity.